peps/pep-0710.rst

```diff
@@ -377,6 +377,9 @@ installing a Python :term:`Distribution Package` is available in the PR to pip
 the ``provenance_url.json`` metadata file for cases when ``direct_url.json`` is not
 created.
 
+Project PDM provided initial support for the ``provenance_url.json`` file,
+following this specification in `pdm-project/pdm#3013`_.
```

Comment:
Rewording this sentence to better fit within the reference implementation section
```suggestion
A reference implementation for supporting the ``provenance_url.json`` file
in PDM exists is available in `pdm-project/pdm#3013`_.
```